src/main/webapp/app/exercises/file-upload/participate/file-upload-submission.component.html

```diff
@@ -49,14 +49,14 @@ <h6>
                 {{ submittedFileExtension | uppercase }}
             </span>
         </div>
-        <div *ngIf="result && result.feedbacks && result.feedbacks.length > 0">
+        <div *ngIf="result && result.feedbacks && result.feedbacks.length && result.feedbacks.length > 0">
```

Comment:
This doesn't need to be `result!.feedbacks!.length! > 0`?

Reply:
I guess that's not necessary as it is checked before that result and result.feedback exist